source/development_process/2_deployment.rst

```diff
@@ -50,16 +50,16 @@ string.
 The Architecture of Chef, Abridged
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Chef has several parts, a central server which holds all of the cookbooks, and
+Chef has several parts: a central server, which holds all of the cookbooks, and
```

Comment:
The comma after cookbooks might be better as a semicolon, now that commas are being used within the list items.